test/entrypoint/config.yml

```diff
@@ -0,0 +1,70 @@
+credentials:
+  users:
+    guest:
+      roles: [super]
+    storage:
+      roles: [sharding]
+      password: "storage"
+
+iproto:
+  listen:
+    - uri: 'unix/:./{{ instance_name }}.iproto'
+  advertise:
+    sharding:
+      login: 'storage'
+      password: 'storage'
+
+sharding:
+  bucket_count: 1234
+  sched_ref_quota: 258
+
+groups:
+  group-001:
+    replicasets:
+      replicaset-001:
+        sharding:
+          roles: [router]
+        roles:
+          - roles.sharded-queue-router
+        database:
+          replicaset_uuid: 'aaaaaaaa-0000-4000-b000-000000000000'
+        instances:
+          queue-router:
+            database:
+              instance_uuid: 'aaaaaaaa-aaaa-4000-b000-000000000001'
+              mode: rw
+          queue-router-1:
+            database:
+              instance_uuid: 'aaaaaaaa-aaaa-4000-b000-000000000002'
+      replicaset-002:
+        sharding:
+          roles: [storage]
+        roles:
+          - roles.sharded-queue-storage
+        database:
```

Comment:
I have never seen it before. What does it do?

Reply:
https://www.tarantool.io/ru/doc/latest/reference/configuration/configuration_reference/#database

In the case we configure `replicaset_uuid`:
https://www.tarantool.io/ru/doc/latest/reference/configuration/configuration_reference/#confval-database.replicaset_uuid

In fact, we could skip it. I did this just to make the configuration as similar as possible to the test Cartridge replicaset.